StudyHub/StudyHub.Common/DTO/User/UserDTO.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace StudyHub.Common.DTO.User;
+
+public class UserDTO
```

Comment:
maybe, if you serch student, than make it StudentDTO and props not nullable?

Reply:
Why rename here when we can use this DTO for all users?

Reply:
because user can be teacher and student and for certain type - certain props.
For instance, you have public string Group { get; set; } = string.Empty; but teacher dont have this property. Thats why I think that it need to be called like StudentDTO